### ISSUE TITLE
Couple Questions

### ISSUE THREAD

**Issue by biggyjdotcom** [scottquach/Canvas-Assignments-Transfer-For-Todoist#46]
To start, im a novice with python and custom scripts in general and im wondering if there's a way customize this code to fit my purposes. First, is there a way to not import overdue assignments? Second, how could i get this program to run regularly, say once a day? Thanks.

**Comment by cjlawson02** [comment]
I'm working on a PR that allows you to ignore assignments you've marked as completed in todoist

**Comment by stacksjb** [comment]
> To start, im a novice with python and custom scripts in general and im wondering if there's a way customize this code to fit my purposes. First, is there a way to not import overdue assignments? Second, how could i get this program to run regularly, say once a day? Thanks.

Sorry, I thought I replied to this!

1) the script imports all assignments marked as unsubmitted in canvas - so even if an assignment is overdue, if it shows that it still is in a "needs submission" workflow state, it will import it. I could see logic for overdue being added in the future if you are interested. 

2) not currently, but this would be done outside of the script (for example, with a chron job.). This is on my list to add functionality for.

**Comment by stacksjb** [comment]
> I'm working on a PR that allows you to ignore assignments you've marked as completed in todoist

Can you help me understand this? Would you be adding logic to look at completed todoist tasks and compare them?

Is there a specific use case for this? Are they null assignments?

**Comment by cjlawson02** [comment]
> Would you be adding logic to look at completed todoist tasks and compare them?

Correct. The two options would be to either reopen todoist tasks that haven't been submitted or to skip importing those tasks.

Right now, I can complete a task but it will repopulate when rerunning the script.

My main use case is optional assignments or group assignments that a teammate submitted on my behalf. Or if I'm a bad student who is skipping assignments.
